scripts/build/companion_libs/050-zlib.sh

```diff
@@ -84,6 +84,7 @@ do_zlib_backend() {
     CT_DoLog EXTRA "Configuring zlib"
 
     CT_DoExecLog CFG                                  \
+    CC="${host}-gcc"                                  \
```

Comment:
Yep, it's contents are:
`#!/usr/local/bin/bash
exec '/usr/local/bin/gcc6' "${@}"`
The configure script for GMP wasn't picking up `x86_64-build_unknown-freebsd11.0-gcc` automatically and instead it built using `cc`. With the (perhaps overly) explicit CC change it behaved as expected. I just also added the CC= stuff to the other scripts to remove any ambiguity.